File: DogAPI/Models/Atendimento.cs
using System;

namespace DogAPI.Models
{
    public class Atendimento
    {
        public int AtendimentoId { get; set; }
        public DateTime DataDeAtendimento { get; set; }
        public Cachorro Pet { get; set; }
        public Veterinario veterinario { get; set; }
        public string Diagnostico { get; set; }
        public string Comentario { get; set; }
        public bool Status { get; set; }
    }
}

File: DogAPIFrontend/DogAPIFrontend/src/app/models/atendimento.ts
export class Atendimento{
  atendimentoId = Number;
  dataDeAtendimento = Date();
  cachorroId = Number;
  veterinarioId = Number;
  diagnostico = String;
  comentario = String;
  pet = {cachorroId : Number};
  veterinario = {veterinarioId : Number}
  horarioDeAtendimento = new Date().getTime();
}

